types/src/front/data_source.ts

```diff
@@ -12,6 +12,8 @@ export const CONNECTOR_PROVIDERS = [
 ] as const;
 export type ConnectorProvider = (typeof CONNECTOR_PROVIDERS)[number];
 
+export type LabsConnectorProvider = ConnectorProvider | "gong";
```

Comment:
Why are we not seeing google here?

Reply:
Because google is already a connector's provider, so it's already there